### ISSUE TITLE
nodeport option

### ISSUE THREAD

**Issue by nehul** [Oteemo/charts#116]
Was going through the service.yaml template for sonarqube 
https://github.com/Oteemo/charts/blob/master/charts/sonarqube/templates/service.yaml

won't it be better to have a node port option if anyone wants to host it using a NodePort on its own network?

**Comment by javajon** [comment]
Agree. While chart allows service to be of type NodePort, the setting of nodePort value is missing. It's a common option for charts with Services to allow setting nodePort. The workaround is to apply a patch after the chart install.
